README.md

```diff
@@ -29,25 +29,30 @@ The awesome list of resources for Microsoft Flight Simulator 2020
 * [UWA's aircraft lighting mods](https://uwajimaya.github.io/FS2020/)
 * [VFR map - in game VFR map](https://flightsim.to/file/3181/fspm-vfr-map)
 * [Pilot2ATC - virtual, AI traffic aware ATC](https://pilot2atc.com/)
+* [Toolbar Pushback - full control over pushback](https://flightsim.to/file/15033/toolbar-pushback)
 
 ## Aircraft / aircraft improvement addons
 
 * [DA40-NGX - Diamond DA40-NG improvement](https://flightsim.to/file/2738/da40-ngx-project)
 * [DA62X - Diamond DA62 improvement](https://flightsim.to/file/2733/da62x-project)
 * [Dassault Dornier Alpha jet](https://flightsim.to/file/7236/dassault-dornier-alpha-jet-dev-version-0-25)
 * [FlyByWire A32NX - Airbus A320 improvement](https://github.com/flybywiresim/a32nx)
+* [Salty Simulations 747 - a modification for the default B748](https://github.com/saltysimulations/salty-747)
 * [Fokker D.21 - FSX conversion](https://flightsim.to/file/3636/fokker-d-21-conversion)
 * [Icon A5 Performance - improvement](https://flightsim.to/file/6806/icon-a5-high-performance-mod)
 * [Savage Carbon - Savage Cub STOL realism improvement](https://flightsim.to/file/3723/savage-carbon-stol-realism-mod)
 * [VL3 Rotax 915](https://github.com/r9r-dev/fs2020-vl3-rotax915)
 * [Working Title improvements for small and medium aircrafts](https://github.com/Working-Title-MSFS-Mods/fspackages)
+* [DC-6](https://pmdg.com/)
 
 ## Multiplayer
 
+* [VATSIM - online flying with ATC](https://vatsim.net)
```

Comment:
Thanks, please leave list in alphabetical order. I look forward to a new pull request.